src/content/en/announcements/2025-04-09-release8.md

```diff
@@ -0,0 +1,42 @@
+# Press Release R8 (for now a headline is missing)
+
+**The internet / Global network (is SCS located anywhere?), 2025-04-09: The Sovereign Cloud Stack community publishes Release 8.** 
+
+While the funding for the Sovereign Cloud Stack (SCS) project by the German Federal Ministry of Economic Affairs and Climate Action (BMWK) ended in December 2024, engaged companies, organizations, and an active open-source community are continuing the maintenance and evolution of the SCS standards, the SCS reference implementation, and the operational skill building. The engineering collaboration results in the publication of the eighth release of the SCS software.
```

Comment:
Isn´t it the nineth release?

Reply:
Yes, it is. However, we have already ignored this subtleness in a previous press release.
So I would continue to ignore it ...

Reply:
@garloff Excellent idea. Yes, Christoph should be able to give a suiteable quote.